data/ecosystems/c/chia.toml

```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
 
 [[repo]]
 url = "https://github.com/zolmine/roboCopy"
+
+[[repo]]
+ url ="https://github.com/somediego/kissmp"
+
+[[repo]]
+ url ="https://github.com/L4bb3rs/Chia-CATS-Sent-to-Bags"
+
+[[repo]]
+ url ="https://github.com/hashgreen/docs.hashgreen.github.io"
+
+[[repo]]
+ url ="https://github.com/SlowestTimelord/slowesttimelord.github.io"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/Dexie.Space.Net"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/Chia-NFT-Minter"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/api"
+
+[[repo]]
+ url ="https://github.com/CryptoLFDM/ChiaRPCWrapper"
+
+[[repo]]
+ url ="https://github.com/Chia-Network/cadt-ui"
+
+[[repo]]
+ url ="https://github.com/greimela/nostr-dex"
+
+[[repo]]
+ url ="https://github.com/gowke/lotus-blockchain"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/CHIA-RPC"
+
+[[repo]]
+ url ="https://github.com/L4bb3rs/Chia-Py-RPC"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-marketplace-admin-dapp"
+
+[[repo]]
+ url ="https://github.com/FireAcademy/telemetry"
```

Comment:
how's this using Chia?

Reply:
FireAcademy is an infra as a service provider for our community.  My understanding was that this project was a data gathering project in that context.